timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TestableTimelockCluster.java

```diff
@@ -54,15 +55,14 @@
 
     private final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-    private final String defaultClient;
+    private final String defaultClient = UUID.randomUUID().toString();
```

Comment:
nit, let's not call this `defaultClient`, since there's no way for you to change it ;)

`client` will do